rpms/build.sh

```diff
@@ -51,11 +52,15 @@ $linux rpmbuild --define "_topdir ${rpm_build_dir}" --define "_threads ${threads
         --define "_debugenabled ${debug}" --target ${architecture_target} \
         -ba ${rpm_build_dir}/SPECS/${package_name}.spec
 
-rpm_file="wazuh-${build_target}-${wazuh_version}-${package_release}.${architecture_target}.rpm"
-pkg_path="${rpm_build_dir}/RPMS/${architecture_target}"
+rpm_file="wazuh-${build_target}-${wazuh_version}-${package_release}*"
```

Comment:
I prefer to add a new variable like `src_rpm_file` with the path to that package and then, use it to compute the checksum and extract it without using `find`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/322/commits/1c0fc432f8a6a01bc925799249c0761244cb4f1f